sdk/communication/azure-communication-chat/azure/communication/chat/_chat_thread_client.py

```diff
@@ -117,6 +119,32 @@ def thread_id(self):
         """
         return self._thread_id
 
+    @distributed_trace
+    def get_properties(
+        self,
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> ChatThreadProperties
+        """Gets a chat thread properties.
```

Comment:
Maybe "Gets the properties of the chat thread."?

Reply:
Changed